Dockerfile.local

```diff
@@ -2,7 +2,7 @@ FROM golang:1.21.3-bullseye AS base
 
 ENV GOCACHE=/go/.go/cache GOPATH=/go/.go/path TZ=Europe/London
 
-RUN GOBIN=/bin go get github.com/cespare/reflex
+RUN GOBIN=/bin go install github.com/cespare/reflex@latest
```

Comment:
The above line needs to be:
RUN GOBIN=/bin go install github.com/cespare/reflex@v0.3.1

This is to prevent any later versions that may be corrupted from injecting evil code into our system.